packages/client/src/components/app/forms/RelationshipField.svelte

```diff
@@ -105,19 +105,29 @@
     }
   }
 
-  $: fetchRows(searchTerm, primaryDisplay)
+  $: fetchRows(searchTerm, primaryDisplay, defaultValue)
 
-  const fetchRows = (searchTerm, primaryDisplay) => {
+  const fetchRows = async (searchTerm, primaryDisplay, defaultVal) => {
     const allRowsFetched =
       $fetch.loaded &&
       !Object.keys($fetch.query?.string || {}).length &&
       !$fetch.hasNextPage
-    // Don't request until we have the primary display
-    if (!allRowsFetched && primaryDisplay) {
-      fetch.update({
-        query: { string: { [primaryDisplay]: searchTerm } },
+    // Don't request until we have the primary display or default value has been fetched
+    if (allRowsFetched || !primaryDisplay) {
+      return
+    }
+    if (defaultVal) {
+      await fetch.update({
+        query: { equal: { _id: defaultValue } },
       })
+      const defaultRow = $fetch.rows?.[0]
+      if (defaultRow) {
+        optionsObj[defaultRow._id] = defaultRow
```

Comment:
Do we need to manually set this? I would have expected the normal reactive statement for `enrichOptions` to have triggered here when this fetch returns, and pulled out the row that comes back here.

Reply:
sadly as there is two operations happening in the one call here, two different fetches, the next one overwrites this one before the reaction it seems (fought with this last night).

Reply:
Interesting! I've seen that behaviour before but never with something async like this. But yea that's no problem then. As long as it only fires when needed then extra duration isn't a problem, since it'll just be on the initial load.